package.json

```diff
@@ -21,11 +21,15 @@
     "@babel/core": "^7.9.0",
     "@babel/preset-env": "^7.9.5",
     "@babel/register": "^7.9.0",
+    "graphql": "^15.3.0",
     "husky": "4.2.5",
     "lint-staged": "10.1.3",
     "mocha": "8.0.1",
     "prettier": "2.0.5"
   },
+  "peerDependencies": {
+    "graphql": "^14.0.0 || ^15.0.0"
```

Comment:
Out of curiosity what made you re-add support for `^14.0.0`?

Also, I've never used `peerDependencies` before, but I wonder if this will break `yarn global add graphql-schema-linter`? 🤔

Reply:
Many graphql libs are still on v14 so if you use them in the same project as this lib you'll potentially get errors for having multiple versions installed.

This lib _should_ work with both versions, but I don't know about prior versions.

Reply:
One of our projects using the linter has this very situation. We can't upgrade graphql to v15 yet because of another dep.

Reply:
> We can't upgrade graphql to v15 yet because of another dep.

Makes sense!

I'm still unsure on whether or not `yarn global add graphql-schema-linter` will stop working because `graphql` would technically be missing. 🤔 For example, imagine my GraphQL API is built in Ruby, so I use `yarn global add graphql-schema-linter` only for linting but otherwise do not have a `package.json` in my project. Not sure if you know?

> This lib _should_ work with both versions, but I don't know about prior versions.

Yeah, I'm fairly certain the linter will work with `^14.0.0`. As an improvement, it might be possible to have `.travis.yml` run the tests against both versions of `graphql`. I've done that for different versions of node, but not sure if it's possible for different versions of packages.

Reply:
Yeah I _think_ if you aren't in a node project you would need to do:
`yarn global add graphql-schema-linter graphql`

Reply:
After reading a bit more about `peerDependencies`, I'm slowly leaning towards your proposal.

Before your PR, I had no idea `peerDependencies` was a thing, which is why I was a bit hesitant. 😄 

> Yeah I think if you aren't in a node project you would need to do

As far as I understand, I think you are correct.

I can't really think of anything else that might break with this change. I think in the `CHANGELOG.md` we'll just need to be clear about users needing to include `graphql` in their `package.json` along side `graphql-schema-linter`. 🤔 

Did you have any thoughts on how we could run the tests against both versions? I haven't looked into that bit yet.

Reply:
I'm not super familiar with running tests against multiple versions of a dependency. Here's one idea that could work (via devDeps and import/env hacks): https://dev.to/joshx/test-your-npm-package-against-multiple-versions-of-its-peer-dependency-34j4